test-jw/package.json

```diff
@@ -1,39 +1,20 @@
 {
-  "name": "my-app",
+  "name": "jwplayer-fandom-test-app",
   "version": "0.1.0",
```

Comment:
Perhaps it's a good opportunity to get rid of this property? I believe it's optional, but if it's left, it would need to be updated.